src/main/java/de/adorsys/keycloak/config/service/RoleImportService.java

```diff
@@ -211,8 +212,21 @@ private void updateRoleIfNeeded(
     ) {
         String roleName = roleToImport.getName();
         RoleRepresentation patchedRole = CloneUtil.patch(existingRole, roleToImport, propertiesWithDependencies);
-        if (roleToImport.getAttributes() != null) {
+
```

Comment:
Looks like method does what it is supposed to. But we need tests to

1. Secure the behavior
2. Secure backward compatibility Role=role -> full

How do we proceed for the removal of existing attributes.